### ISSUE TITLE
Group Normalization

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#694]
## 一言でいうと

Batch Normalizationはバッチのサイズが小さくなるとエラー率が高くなってしまうため、チャンネルをグループに分けてそれ毎に正規化を行おうという話(論文中の図の解説がわかりやすい)。チャンネルは相互に影響しあっているはずなので、グループでまとめてもいいはずとのこと

![image](https://user-images.githubusercontent.com/544269/38122241-c64ce79c-340e-11e8-8d24-313d76a8f200.png)

### 論文リンク

https://arxiv.org/abs/1803.08494

### 著者/所属機関

Yuxin Wu, Kaiming He
* Facebook AI Research (FAIR)

### 投稿日付(yyyy/MM/dd)

2018/3/22

**Comment by icoxfog417** [comment]
PyTorchはmasterにmerge
https://github.com/pytorch/pytorch/pull/5968
